src/file.ts

```diff
@@ -2295,11 +2295,14 @@ class File extends ServiceObject<File> {
    *     'v2' or 'v4.
    * @param {string} [config.cname] The cname for this bucket, i.e.,
    *     "https://cdn.example.com".
-   * @param {string} [config.contentMd5] The MD5 digest value in base64. If you
-   *     provide this, the client must provide this HTTP header with this same
-   *     value in its request.
-   * @param {string} [config.contentType] If you provide this value, the client
-   *     must provide this HTTP header set to the same value.
+   * @param {string} [config.contentMd5] The MD5 digest value in base64. Just like
+   *     if you provide this, the client must provide this HTTP header with this same
```

Comment:
The MD5 digest value in base64. If you provide this, the client must provide this HTTP header the same value in its request, so if this parameter is not provided here, the client must not provide ...

Reply:
(same comment for the next one too.) If you provide this, the client
   *     must provide this HTTP header with this same value in its request, so  if
   *     this parameter is not provided here, the client must not provide any value
   *     for this HTTP header in its request.